notebooks/306-nncf-with-nlp-huggingface-transformers/bert_mrpc_int8.yml

```diff
@@ -0,0 +1,39 @@
+models:
+  - name: bert-mrpc
+    launchers:
+      - framework: dlsdk
+        device: CPU
+        adapter:
+            type: classification
+            num_classes: 2
+        inputs:
+          - name: "result.1"
```

Comment:
is it possible to export int8 model with preserving input names like in original?